vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -376,23 +415,42 @@ export default class BaseAddress extends Mixins(ValidationMixin) {
   }
 
   /**
-   * Updates the address data after the user chooses a suggested address.
+   * Callback to update the address data after the user chooses a suggested address.
    *
    * @param address the data object returned by the AddressComplete Retrieve API.
    */
   private addressCompletePopulate (address: object): void {
```

Comment:
This is beyond the scope of this PR, but just noticed we are receiving a plain object back from this call (presumably from Canada Post API), and then indexing string properties without any schema check.  I'm wondering if we should create another ticket to make this a bit more robust by validating the returned object first, and if validation fails (because of a schema change), we can handle it a bit more gracefully.  Just a thought, and by no means should that be included here :)

Reply:
That's not a bad idea. The Canada Post "integration" (and I use that word loosely) does not appear to work as documented, so we could likely improve how it functions among other things. I agree this is the subject of another ticket. This ticket was to ensure country was valid (as picked from a list) and I included a number of UI features per the UxPin.